view/frontend/templates/button_product_page.phtml

```diff
@@ -193,15 +193,22 @@ if (!$block->isSupportableType()) return;
                 var options = {};
                 formData = new FormData(document.getElementById("product_addtocart_form"));
                 formData.forEach(function(value, key){
-                    matchResult = key.match(/super_attribute\[(\d+)\]/);
-                    if (matchResult) {
-                        if (!('super_attribute' in options)) {
-                            options['super_attribute'] = {};
+
+                    var compositeAttributtes = ['super_attribute', 'options'];
```

Comment:
Could we move this var above forEach loop? Just to not assign on every iteration.
And nit: typo in word "Attributes"

Reply:
@vitaliyreznikov Done. Can you take another look?